packages/expo-router/src/fork/useLinking.ts

```diff
@@ -149,12 +149,7 @@ export default function useLinking(
     getActionFromStateRef.current = getActionFromState;
   });
 
-  /* Start of fork. Source: https://github.com/react-navigation/react-navigation/blob/13d4aa270b301faf07960b4cd861ffc91e9b2c46/packages/native/src/useLinking.tsx#L142 */
-  // ServerContext is used inside ServerContainer to set the location during SSR: https://github.com/react-navigation/react-navigation/blob/13d4aa270b301faf07960b4cd861ffc91e9b2c46/packages/native/src/ServerContainer.tsx#L50-L54
-  // Expo Router uses the `initialLocation` prop to set the initial location during SSR:
-  const location = React.useContext(ServerLocationContext);
-  const server = { location };
-  /* End of fork */
+  const server = React.useContext(ServerContext);
```

Comment:
I've updated the context to exactly match upstream. https://github.com/expo/expo/pull/27757/files#r1531467252

We might not need this fork anymore, but that's for a different PR